qiskit/algorithms/__init__.py

```diff
@@ -148,6 +148,7 @@
     PVQDResult
     TimeEvolutionResult
     TimeEvolutionProblem
+    TrotterQRTE
```

Comment:
Which one does this link to - the old one. The reason it was not included here, was the old one still exists at this level. Its the same issue as VQE where its the pre-existing on at this level. It is here in [time_evolvers.trotterization](https://qiskit.org/documentation/stubs/qiskit.algorithms.time_evolvers.trotterization.html) ref. Yes it might be nice at this level but not until the prior one is removed can we think about allowing this.

Reply:
Thanks for the info, removed in 2f3838b!